new-installer/internal/config/config.go

```diff
@@ -43,17 +43,25 @@ type OrchInstallerRuntimeState struct {
 	StateBucketState string `yaml:"stateBucketState"` // The state S3 bucket Terraform state
 	// Move runtime state here?
 	AWS struct {
-		KubeConfig               string   `yaml:"kubeConfig"`
-		CacheRegistry            string   `yaml:"cacheRegistry"`
-		VPCID                    string   `yaml:"vpcID"`
-		PublicSubnetIDs          []string `yaml:"publicSubnetIDs"`
-		PrivateSubnetIDs         []string `yaml:"privateSubnetIDs"`
-		JumpHostIP               string   `yaml:"jumpHostIP"`
-		JumpHostSSHKeyPublicKey  string   `yaml:"jumpHostSSHPublicKey"`
-		JumpHostSSHKeyPrivateKey string   `yaml:"jumpHostSSHPrivateKey"`
-		EFSFileSystemID          string   `yaml:"efsFileSystemID"`
-		EKSOIDCIssuer            string   `yaml:"eksOIDCIssuer"`
-		ACMCertArn               string   `yaml:"acmCertArn"`
+		KubeConfig                string   `yaml:"kubeConfig"`
+		CacheRegistry             string   `yaml:"cacheRegistry"`
+		VPCID                     string   `yaml:"vpcID"`
+		PublicSubnetIDs           []string `yaml:"publicSubnetIDs"`
+		PrivateSubnetIDs          []string `yaml:"privateSubnetIDs"`
+		JumpHostIP                string   `yaml:"jumpHostIP"`
+		JumpHostSSHKeyPublicKey   string   `yaml:"jumpHostSSHPublicKey"`
+		JumpHostSSHKeyPrivateKey  string   `yaml:"jumpHostSSHPrivateKey"`
+		EFSFileSystemID           string   `yaml:"efsFileSystemID"`
+		EKSOIDCIssuer             string   `yaml:"eksOIDCIssuer"`
+		ACMCertARN                string   `yaml:"acmCertARN"`
+		TraefikLBARN              string   `yaml:"traefikLBARN"`
```

Comment:
I understand that we need TG ARN in later steps when TGB is created, but when do we need LB ARN and DNS name?